extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -132,17 +132,25 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 
 		const selectedSubscriptionQuickPickItems = await vscode.window.showQuickPick(subscriptionQuickPickItems, { canPickMany: true });
 		if (selectedSubscriptionQuickPickItems && selectedSubscriptionQuickPickItems.length > 0) {
-			await tree.refresh(node, false);
+			for (const tree of trees) {
```

Comment:
Should we do this in a Promise.all() to refresh the trees in parallel?

Reply:
commented about this in the other comment similar to this.